### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by dumez-k**

### CODE REVIEW

File: resource/check.go [hunk 1]
```diff
@@ -15,7 +16,7 @@ import (
 )
 
 func Check(clientset versioned.Interface, source Source, version oc.Version, env oc.Environment, logger Logger) ([]oc.Version, error) {
-	buildList, err := clientset.KpackV1alpha1().Builds(source.Namespace).List(metav1.ListOptions{
+	buildList, err := clientset.KpackV1alpha1().Builds(source.Namespace).List(context.TODO(), metav1.ListOptions{
```
Comment (matthewmcnew) [inline resource/check.go:19]: Can we provide the context from main?

File: go.mod [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
-	k8s.io/api v0.17.6
-	k8s.io/apimachinery v0.17.6
+	github.com/stretchr/testify v1.7.0
+	k8s.io/api v0.19.7
+	k8s.io/apimachinery v0.19.7
 	k8s.io/client-go v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible
 )
 
-replace k8s.io/client-go => k8s.io/client-go v0.17.5
+replace k8s.io/client-go => k8s.io/client-go v0.19.7
```
[the remark this answers is not in the document]
Comment (dumez-k) [reply go.mod:17]: nope!

File: resource/check.go (hunk 1 above)
Comment (dumez-k) [reply resource/check.go:19]: sure thing

**Comment by dumez-k** [comment]
addressed your comments @matthewmcnew

File: cmd/resource/main.go [hunk 3]
```diff
@@ -32,6 +33,8 @@ func main() {
 type concourseResource struct{}
 
 func (concourseResource) Check(ocSource ofcourse.Source, version ofcourse.Version, env ofcourse.Environment, logger *ofcourse.Logger) ([]ofcourse.Version, error) {
+	ctx := context.Background()
```
Comment (matthewmcnew) [inline cmd/resource/main.go:36]: Nice!

File: resource/check.go [hunk 4]
```diff
@@ -14,8 +15,8 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func Check(clientset versioned.Interface, source Source, version oc.Version, env oc.Environment, logger Logger) ([]oc.Version, error) {
-	buildList, err := clientset.KpackV1alpha1().Builds(source.Namespace).List(metav1.ListOptions{
+func Check(clientset versioned.Interface, source Source, version oc.Version, env oc.Environment, logger Logger, ctx context.Context) ([]oc.Version, error) {
```
Comment (matthewmcnew) [inline resource/check.go:18]: Could we make the ctx the first argument just for consistency?

File: resource/in.go [hunk 5]
```diff
@@ -25,7 +27,7 @@ func (in *In) In(outDir string, source Source, params oc.Params, version oc.Vers
 		return nil, nil, err
 	}
 
-	buildList, err := in.Clientset.KpackV1alpha1().Builds(source.Namespace).List(metav1.ListOptions{
+	buildList, err := in.Clientset.KpackV1alpha1().Builds(source.Namespace).List(context.TODO(), metav1.ListOptions{
```
Comment (matthewmcnew) [inline resource/in.go:30]: Could we pass in Context from main everywhere in resource/*.go ?

Comment (dumez-k) [reply resource/in.go]: 👍

**Review by tomkennedy513 — APPROVED** [review]

**Comment by matthewmcnew** [comment]
So good @dumez-k !

**Review by matthewmcnew — APPROVED** [review]

**Pull request merged**